.github/workflows/ci.yml

```
@@ -12,6 +12,9 @@ jobs:
   verify:
     name: Verify
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: .   
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)